tools/ci/truffle_test

```diff
@@ -16,8 +16,10 @@ yarn workspace @chainlink/box run lint
 #   `cd evm/v0.5 && yarn build && slither --triage-mode .`
 
 yarn workspace chainlinkv0.5 build
+yarn workspace chainlinkv0.6 build
 
 # These should be merged into a global test command
 yarn workspace chainlink test
 yarn workspace chainlinkv0.5 test
+yarn workspace chainlinkv0.6 test
 yarn workspace @chainlink/box test
```

Comment:
How did this even get through without a newline?

Reply:
Not sure what you mean, tests dont run linters. Even then, those missing new lines have always existed, I copied the files over directly

Reply:
5f5d7a3